CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributer's Guide
+
+We welcome contributions from the community and have gathered guidelines 
+here to help you get started.
+
+## Discussion
+
+While not absolutely required, it is encouraged that you first open an issue 
+for any bug or feature request. This allows discussion on the proper course of
+action to take before coding begins.
+
+## Building
+
+To build from source you first need to `npm install`. You will also need a
+version of TypeScript available, either globally or install locally with
+`npm install typescript`. Then run `node build`.
```

Comment:
Yea I'm good with changing to `npm run build`. I considered doing that originally. Not sure why I didn't. Regarding the dev dependency, that would probably work as long as it doesn't mess up the [travis build](https://github.com/TypeStrong/ts-loader/blob/master/.travis.yml).